pages/faq.tsx

```diff
@@ -1,16 +1,15 @@
-import Head from 'next/head'
-
 import { Box, Heading, Link, Text } from '@chakra-ui/react'
 import styles from '../styles/FaqPage.module.css'
 import ContentBox from '../components/common/ContentBox'
+import HeadTags from '../components/common/HeadTags'
 
 export default function FAQ() {
   return (
     <div className={styles.faq}>
-      <Head>
-        <title>FAQ - My COVID Story</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+      <HeadTags
+        title="FAQ"
+        description="Frequently Asked Questions about how to share your story of COVID-19"
```

Comment:
Definitely up for some edits on this copy.

Reply:
title="Frequently Asked Questions | MyCovidStory.ca"
description="View our list of Frequently Asked Questions to learn more about how the site began, moderation guidelines and how we protect your privacy."

Reply:
I don't think I did this whole making changes and contributions thing right but there's some copy for ya😅